.lintstagedrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "*.{js,jsx}": ["eslint --fix"],
+  "*.{json,md}": ["prettier --write"]
+}
```

Comment:
I tried changing a js file so a code line goes over 80 chars and Prettier did reformat it, however the commit didn't fail. We need to figure out a way to stop the commit if any tool makes changes to the files ([`--bail` behavior](https://github.com/azz/pretty-quick#--bail) of `pretty-quick`). (Would also be good to keep [`--verbose`](https://github.com/azz/pretty-quick#--verbose) too).

> From https://github.com/iterative/dvc.org/pull/790#discussion_r346641520

Reply:
The commit didn't fail because the format issues have been fixed automatically using `eslint --fix`. Do you want the developer to fix the format issues manually?

Reply:
They were fixed but not staged because we don't want automatic staging of fixes. (At least that's how it worked with `pretty-quick`.) This allows the developer to review the automatic formatting with `git diff` before adding them manually and trying to commit again. I wonder if this will be possible with `lint-staged`... Maybe this is helpful: https://medium.com/hackernoon/announcing-lint-staged-with-support-for-partially-staged-files-abc24a40d3ff ?

Re `--verbose`: perhaps `lint-staged --debug`? See https://github.com/okonet/lint-staged#command-line-flags

Also, probably simpler to put this [in package.json](https://github.com/okonet/lint-staged#packagejson-example) instead of a standalone config file?

Reply:
Thanks for moving to package.json!

> I wonder if this will be possible...

Possibly also related/useful: okonet/lint-staged/issues/550, https://github.com/okonet/lint-staged/issues/594#issuecomment-474304561. I also opened okonet/lint-staged/issues/730, let's see what they think.

Reply:
Resolving in favor of https://github.com/iterative/dvc.org/pull/790#issuecomment-554811425.